client/reader/stream/post-lifecycle.jsx

```diff
@@ -54,7 +54,7 @@ class PostLifecycle extends Component {
 					<PostPlaceholder />
 				</Fragment>
 			);
-		} else if ( post._state === 'error' ) {
+		} else if ( post.is_error ) {
```

Comment:
The `is_error` and `error` props are set here: https://github.com/Automattic/wp-calypso/blob/0fe5ebb3e11280fd8326f7d3ec57aaf79ffd1ae1/client/state/reader/posts/actions.js#L151